socorro/external/fs/crashstorage.py

```diff
@@ -214,7 +214,9 @@ def get_raw_dump(self, crash_id, name=None):
         ) as f:
             return f.read()
 
-    def get_dumps_as_files(self, crash_id):
+    def get_dumps_as_files(self, crash_id, tmpdir):
+        # NOTE(willkg): We don't need to use tmpdir here because the files are already
```

Comment:
The file system already has the files on the file system where the stackwalker (and other things) can access them. We don't need to copy them to another place on the file system. We also don't want to delete them when processing of this crash is done. Given that, we can ignore the temporary directory.